src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/typer.ts

```diff
@@ -10,21 +10,28 @@ export interface ITidligereInnvilgetVedtak {
     harTidligereOvergangsstønad: boolean;
     harTidligereBarnetilsyn: boolean;
     harTidligereSkolepenger: boolean;
-    periodeHistorikkOvergangsstønad: IGrunnlagsdataPeriodeHistorikk[];
+    periodeHistorikkOvergangsstønad: IGrunnlagsdataPeriodeHistorikkOvergangsstønad[];
+    periodeHistorikkBarnetilsyn: IGrunnlagsdataPeriodeHistorikkBarnetilsyn[];
 }
 
-export interface IGrunnlagsdataPeriodeHistorikk {
+export interface IGrunnlagsdataPeriodeHistorikkOvergangsstønad {
     antallMåneder: number;
     antallMånederUtenBeløp: number;
     vedtaksperiodeType: EPeriodetype;
     fom: string;
     tom: string;
 }
-
+export interface IGrunnlagsdataPeriodeHistorikkBarnetilsyn {
```

Comment:
Er dette en `GrunnlagsdataPeriodeHistorikkBarnetilsyn` eller er dette egentlig bare `FraTilDato`

Reply:
Dette er egentlig IGrunnlagsdataPeriodeHistorikkBarnetilsyn. Kommer også felt for overlapp senere